apps/src/sites/studio/pages/levels/levelGroup.js

```diff
@@ -53,10 +51,9 @@ window.initLevelGroup = function (levelCount, currentPage, lastAttempt) {
         app: subLevel.getAppName(),
         allowMultipleSends: true,
         level: subLevelId,
-        result: subLevelResult,
-        pass: subLevelResult,
-        testResult: testResult,
-        submitted: submitted,
+        result: true,
+        testResult: TestResults.UNVALIDATED_SUBLEVEL,
+        submitted: subLevelResult.submitted || false,
         onComplete: handleSublevelComplete
```

Comment:
https://github.com/code-dot-org/code-dot-org/pull/12694 Is also worth looking at if you're interested in my investigation of all the different ways we use the report object (spoiler alert: its a bit of a mess).